src/bepasty/utils/date_funcs.py

```diff
@@ -1,17 +1,24 @@
 import time
 from flask import current_app
+from werkzeug.exceptions import BadRequest
 
 from ..constants import FOREVER, TIMESTAMP_MAX_LIFE
 
 
 def get_maxlife(data, underscore):
-    unit_key = 'maxlife_unit' if underscore else 'maxlife-unit'
-    unit_default = 'MONTH'
+    unit_key = 'maxlife_unit' if underscore else 'Maxlife-Unit'
+    unit_default = 'MONTHS'
     unit = data.get(unit_key, unit_default).upper()
-    value_key = 'maxlife_value' if underscore else 'maxlife-value'
+    value_key = 'maxlife_value' if underscore else 'Maxlife-Value'
```

Comment:
if maxlife-unit and maxlife-value were not broken, do not "fix" them.

Reply:
ok, if you think so. removed